ecs-agent/netlib/platform/common_linux.go

```diff
@@ -38,6 +41,7 @@ import (
 	"github.com/aws/amazon-ecs-agent/ecs-agent/volume"
 
 	"github.com/aws/aws-sdk-go/aws"
+	currentCNITypes "github.com/containernetworking/cni/pkg/types/100"
```

Comment:
[non blocking comment] nit - golang pkg name prefer low cases without caps and underscores. so probably currecntcnitypes or cnitypes?

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.
https://go.dev/blog/package-names

Reply:
Makes sense.